### DISCUSSION TITLE
updateproblem DownloadHelper 7 to Video DownloadHelper 8.2.0.24

### DISCUSSION THREAD

**Discussion by Yannick-Kniebs** [aclap-dev/video-downloadhelper#154] [Help and Q&A]
### Your question

Hello everyone
Since I updated DownloadHelper 7 to version Video DownloadHelper 8.2.0.24, it no longer downloads the video files like it used to. It used to always display the title and automatically download it as an MP4. why not anymore?? can someone help me please?

This is what the downloaded files now look like 
![Image2](https://github.com/aclap-dev/video-downloadhelper/assets/160361725/7d29d65c-9671-4a55-96d5-973e86ac4fa0)

**Comment by milsabords1** [comment]
same problem since FEBRUARY 15, 2024. it has always worked well, and since yesterday it no longer works very well

W7  64 BITS  FIREFOX  APP COMPAGNON Installée

**Comment by GrampaWildWilly** [comment]
Paul has said he's working on correcting this problem.  I don't work for VDH but I am curious how this happens.  Would you mind posting a series of screenshots showing your dialog with VDH: The initial VDH menu, which variant you pick, what the file selection dialog proposes to you.  Something along those lines.

**Reply by Yannick-Kniebs** [reply]
Thanks for the answer, grandpa.
Here is the process when I download a video from a stremer.

1. I click on the provider “streamtape” at the bottom "streampage" 
![1](https://github.com/aclap-dev/video-downloadhelper/assets/160361725/94395022-b1f6-4824-a4f1-59045b7619a4)

2. Then I go to the VDH logo at the top right and select the video.
![2](https://github.com/aclap-dev/video-downloadhelper/assets/160361725/be3e8d4c-ebc9-49a9-8921-7471a5cd82a3)

3. then I select herunterladen (download).
![3](https://github.com/aclap-dev/video-downloadhelper/assets/160361725/1b281d74-0db0-4433-afaf-e364b63b7bda)

4. and then he saves the video. without the title.
![4](https://github.com/aclap-dev/video-downloadhelper/assets/160361725/4d190aae-e9f2-4138-9d5b-7533f8835417)


**Before the update I always did it like that for years and it memorized the entire title of the video. why doesn't this work anymore?**

**Reply by GrampaWildWilly** [reply]
Thank you very much for that.

Now that is interesting.  And remember.  I don't work for VDH.  I'm just another user.  I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3155, Firefox 122.0.1 64-bit, licensed VDH 8.2.1.0a1 beta, CoApp 2.0.17.  You have not supplied all of that information & you really should.  I went to your page & did everything you did.  VDH proposed this file name for me:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/6aa8f0c0-d541-4069-8ce4-6e91fe34af8f)

Nothing like what you got.  Also, your file name says .mkv.mp4.  That is totally bizarre.

Also, you should know there is a new VDH setting:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/8ecf343f-77e1-42e4-a125-14e12419caae)

Using that setting, VDH proposes this for me:

![#03](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/46a6ed91-7a38-407f-a456-3f3374b8462d)

It would be interesting to see whether the new setting changes things for you.  Personally, I have never liked the file selection dialog in VDH.  It required too much mouse interaction.  Plus it's kind of like reinventing the wheel.  This new way is so much better because, (a) it uses what's already built into the operating system & (b) I can operate this using just the keyboard, no mousing involved.

Whatever the case, I did launch the download & it looks like this:

![#04](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/b0712ac4-debb-42b7-832f-4ca6e3b05d8e)

That is a horrifically slow download speed.  I will probably just interrupt it as soon as I finish posting this.  Other people in here have been complaining about slow downloads.  I have been seeing slow downloads on a couple of the sites other users have been posting about.  But I am not seeing problems on, for example, YouTube.  So it would be hard to blame VDH for the slow download speed.  I think it's the serving web site.  They are offering free content so they believe they don't have to spend a lot of money on hardware to make things download faster.  They just pocket the advertising fees they get without investing any of that in making their site better.

The bottom line is I am not seeing the weird effect you are.  If you can imagine Paul sitting at home looking at this, he's seeing one user who is having a problem & another who isn't.  He's got to be asking himself, "How can I write code for these guys?"  I kind of wish I had seen the problem you are seeing.  That would make it easy for Paul.  But the reality is it's not happening to me.  I don't know where this leaves us.

**Comment by milsabords1** [comment]
l'appli compagnon est : VdhCoApp 2.0.17

par ailleurs j'ai mis en téléchargement deux vidéos hier avant d'aller dormir et les videos ont été téléchargées. mais je ne sais pas combien de temps ça a pris.

l'appli semble fonctionner differemment. dans le choix du téléchargeur j'ai coché "demander" , et en fait il ne dema,de pas quand je clique sur une vidéo que je veux télécharger , il n' affiche rien , et parfois dans le dossier de destination rien ne s'affiche, mais là , oui il y a eu les fichiers temporaires, puis les fichiers définitifs.  je viens de lancer deux nouveaux téléchargements, on va voir.

**Comment by milsabords1** [comment]
la mise a jour comporte probablement une erreur qui a créé un bug

**Reply by GrampaWildWilly** [reply]
Tu devrais prendre comme exemple ce que nous avons fait ci-haut.  Fais des captures d'écran.  Et décris pour nous ton environnement d'ordi, tel que j'ai fait ou selon ça: https://github.com/aclap-dev/video-downloadhelper/wiki/I-need-help.

**Comment by Yannick-Kniebs** [comment]
I have meanwhile uninstalled and reinstalled the download helper twice. Suddenly it works the way it always worked. Thank you anyway for your help.

![s960_thank_you_sticky_note](https://github.com/aclap-dev/video-downloadhelper/assets/160361725/b95d52b2-3eb1-40ab-8946-cd6559a94767)

**Reply by milsabords1** [reply]
GOOD  !  vous etes sur W 7 ?

**Reply by Yannick-Kniebs** [reply]
Windows 10

![Image3](https://github.com/aclap-dev/video-downloadhelper/assets/160361725/1190880b-29d3-4cc4-ad30-136259133462)
